SUPPORT.md

```diff
@@ -0,0 +1,61 @@
+
+To help us resolve your issue as soon as possible, please follow these steps:
+
+
+<a id="orge8594a5"></a>
+
+# Check for already opened issues:
+
+-   In `google-cloud-python`: <https://github.com/googleapis/google-cloud-python/issues>
+-   In Stack Overflow: <https://stackoverflow.com/questions/tagged/google-cloud-platform+python>
+
+
+<a id="org4510fe1"></a>
+
+# *BEFORE* reporting an issue here
+
+-   Please determine whether it's a client library issue or an underlying API service issue.
+    -   **Bugs**: Try reproducing the issue without the client library; if you can, then it's an API issue.
+    -   **Features**: Determine whether you want something different about the data or how it's changed in Google services (rather than how you access or manipulate it locally); if so, it's an API issue.
+-   To make these determinations, visit [Google APIs Explorer](https://developers.google.com/apis-explorer):
+    1.  Find your API on the [APIs Explorer list](https://developers.google.com/apis-explorer), and follow the link.
+    2.  Click on the API method corresponding to your issue.
+    3.  In the page that appears:
+        -   You can use the APIs Explorer sidebar (“Try this Method”), fill in your request parameters, and see whether your request succeeds or causes the same error you were getting initially.
+        -   Alternatively, you can use the `HTTP Request` listed on that page and any information on the `Request body` to construct a request you can send from the command line using a tool like `curl`.
+    4.  If your request in step 3 above succeeds and is as you expect, then it's likely the resolution centers on the client library; [file a client library issue](#org9c34cc3) as detailed below. If your request failed or returned unexpected results, then it's likely the resolution centers on the API service; [file an API service issue](#org93f7174) instead, as detailed below.
+
+
+<a id="org93f7174"></a>
+
+# Filing an API service issue
+
+*(if the issue DID re-occur [above](#org4510fe1) without using the client library)*
+
+Use the appropriate API service issue tracker. The maintainers of this repo are not experts or contributors on individual API services; we work on generating usable and idiomatic client libraries for many APIs.
+
+1.  Consider using [Google Cloud Customer Care](https://cloud.google.com/support/?hl=en) (paid) to get more dedicated support for your issue. Otherwise, continue with the steps below.
+2.  Find your API in [the client list at the top of this repository](https://github.com/googleapis/google-cloud-python/tree/main?tab=readme-ov-file#libraries).
+3.  Check the “Issue Tracker” column to see whether someone else has reported the same issue. If they have, the filed issue may have some useful information; feel free to add more details or context of your own.
```

Comment:
Should we add a note that folks can file API issues in the corresponding client library repository in the case where there is no API specific issue tracker?